src/components/Home.js

```diff
@@ -1,13 +1,26 @@
-import { useSelector } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
 import Card from './Card';
 import AddForm from './AddForm';
+import { fetchBook } from '../redux/books/bookSlice';
 
 const Home = () => {
-  const { books } = useSelector((state) => state.books);
+  const dispatch = useDispatch();
+  const { books, isLoading, error } = useSelector((state) => state.books);
 
+  useEffect(() => {
+    dispatch(fetchBook());
+  }, [dispatch]);
+
+  if (isLoading) {
+    return (<span>Loading...</span>);
+  } if (error) {
+    return (<span>There is an error rendering your content</span>);
+  }
   return (
     <section className="container bg">
-      {books.map((items) => <Card key={items.item_id} action={items.category} title={items.title} author={items.author} percent="64" chapter="Chapter 17" id={items.item_id} />)}
+
+      {books.map((items) => (<Card key={items.item_id} action={items.category} title={items.title} author={items.author} percent="64" chapter="Chapter 17" id={items.item_id} />))}
       <div className="horizontal-rules" />
       <div>
         <AddForm />
```

Comment:
- [ ] It would be better to remove the spaces that I indicated in the following screenshot it doesn't make sense to have a website with spaces from the top, left, and right  just think about it
 the nav bar should take full width and it should be no spaces in the edges 
 
<img width="1440" alt="Screenshot 2023-05-12 at 19 01 40" src="https://github.com/boniface193/bookstore/assets/61790587/d6efc66b-4824-4448-a11c-21836bd0a3cd">